packages/beacon-node/src/metrics/metrics/beacon.ts

```diff
@@ -58,6 +58,66 @@ export function createBeaconMetrics(register: RegistryMetricCreator) {
 
     // Non-spec'ed
 
+    // Finalized block and proposal stats
+    allValidators: {
```

Comment:
Just reading the metric name `all_validators_expected_count` or `allValidators.expected` does not make it obvious that this is about blocks